examples/dgmulti_2d/elixir_mhd_reflective_wall.jl

```diff
@@ -8,70 +8,73 @@ using LinearAlgebra: norm, dot # for use in the MHD boundary condition
-  # pressure perturbation in a vertically magnetized field on the domain [-1, 1]^2
+    # pressure perturbation in a vertically magnetized field on the domain [-1, 1]^2
 
-  r2 = (x[1] + 0.25)^2 + (x[2] + 0.25)^2
+    r2 = (x[1] + 0.25)^2 + (x[2] + 0.25)^2
 
-  rho = 1.0
-  v1  = 0.0
-  v2  = 0.0
-  v3  = 0.0
-  p   = 1 + 0.5 * exp(-100 * r2)
+    rho = 1.0
+    v1 = 0.0
+    v2 = 0.0
+    v3 = 0.0
+    p = 1 + 0.5 * exp(-100 * r2)
 
-  # the pressure and magnetic field are chosen to be strongly
-  # magnetized, such that p / ||B||^2 ≈ 0.01.
-  B1  = 0.0
-  B2  = 40.0 / sqrt(4.0 * pi)
-  B3  = 0.0
+    # the pressure and magnetic field are chosen to be strongly
+    # magnetized, such that p / ||B||^2 ≈ 0.01.
+    B1 = 0.0
+    B2 = 40.0 / sqrt(4.0 * pi)
+    B3 = 0.0
 
-  psi = 0.0
-  return prim2cons(SVector(rho, v1, v2, v3, p, B1, B2, B3, psi), equations)
+    psi = 0.0
+    return prim2cons(SVector(rho, v1, v2, v3, p, B1, B2, B3, psi), equations)
 end
 initial_condition = initial_condition_perturbation
 
 surface_flux = (flux_lax_friedrichs, flux_nonconservative_powell)
-volume_flux  = (flux_hindenlang_gassner, flux_nonconservative_powell)
+volume_flux = (flux_hindenlang_gassner, flux_nonconservative_powell)
 
-solver = DGMulti(polydeg=3, element_type = Quad(), approximation_type = GaussSBP(),
+solver = DGMulti(polydeg = 3, element_type = Quad(), approximation_type = GaussSBP(),
                  surface_integral = SurfaceIntegralWeakForm(surface_flux),
                  volume_integral = VolumeIntegralFluxDifferencing(volume_flux))
 
-x_neg(x, tol=50*eps()) = abs(x[1] + 1) < tol
-x_pos(x, tol=50*eps()) = abs(x[1] - 1) < tol
-y_neg(x, tol=50*eps()) = abs(x[2] + 1) < tol
-y_pos(x, tol=50*eps()) = abs(x[2] - 1) < tol
+x_neg(x, tol = 50 * eps()) = abs(x[1] + 1) < tol
+x_pos(x, tol = 50 * eps()) = abs(x[1] - 1) < tol
+y_neg(x, tol = 50 * eps()) = abs(x[2] + 1) < tol
+y_pos(x, tol = 50 * eps()) = abs(x[2] - 1) < tol
 is_on_boundary = Dict(:x_neg => x_neg, :x_pos => x_pos, :y_neg => y_neg, :y_pos => y_pos)
 
 cells_per_dimension = (16, 16)
-mesh = DGMultiMesh(solver, cells_per_dimension; periodicity=(false, false), is_on_boundary)
+mesh = DGMultiMesh(solver, cells_per_dimension; periodicity = (false, false),
+                   is_on_boundary)
 
 # Create a "reflective-like" boundary condition by mirroring the velocity but leaving the magnetic field alone.
 # Note that this boundary condition is probably not entropy stable.
-function boundary_condition_velocity_slip_wall(u_inner, normal_direction::AbstractVector, x, t,
-                                               surface_flux_function, equations::IdealGlmMhdEquations2D)
+function boundary_condition_velocity_slip_wall(u_inner, normal_direction::AbstractVector, x,
+                                               t,
+                                               surface_flux_function,
+                                               equations::IdealGlmMhdEquations2D)
 
```

Comment:
In function `boundary_condition_velocity_slip_wall` would we rather want the arguments `t` and `surface_flux_function` in different lines?

Reply:
It looks odd to me, I'd prefer `x, t` on the same line.

Reply:
It's fixable, e.g., see 6b775b37. Can you maybe directly fix this wherever you find it in this PR?